runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/GrpcWindmillServer.java

```diff
@@ -632,6 +632,8 @@ private static long uniqueId() {
     // The following should be protected by synchronizing on this, except for
     // the atomics which may be read atomically for status pages.
     private StreamObserver<RequestT> requestObserver;
+    // Indicates if the current stream in requestObserver is client closed
```

Comment:
Could you clarify the language here? I'm not sure what "is client closed" means

Reply:
done, simplified it.